LDAR_Sim/src/programs/method.py

```diff
@@ -100,12 +99,12 @@ def initialize_crews(self, crews, sites: "list[Site]") -> None:
 
     def initialize_cost_tracking(self, cost_properties: dict[str, float]):
         self.upfront_cost = cost_properties[pdc.Method_Params.UPFRONT] * self._crews
-        if cost_properties[pdc.Method_Params.PER_SITE] > 0:
-            self.cost_type = self.PER_SITE_COST
-            self.cost = cost_properties[pdc.Method_Params.PER_SITE]
-        elif cost_properties[pdc.Method_Params.PER_DAY] > 0:
+        if cost_properties[pdc.Method_Params.PER_DAY] > 0:
             self.cost_type = self.PER_DAY_COST
             self.cost = cost_properties[pdc.Method_Params.PER_DAY]
+        elif cost_properties[pdc.Method_Params.PER_SITE] > 0:
+            self.cost_type = self.PER_SITE_COST
+            self.cost = cost_properties[pdc.Method_Params.PER_SITE]
         else:
             self.cost_type = self.PER_SITE_COST
```

Comment:
Should we error out here if we get here with stationary?

Reply:
Handled it such that the simulation should still run when no costs are given.